build.ps1

```diff
@@ -40,11 +40,38 @@ if( Test-Path -Path ('env:APPVEYOR') )
 
     $buildInfo = '+{0}.{1}.{2}' -f $env:APPVEYOR_BUILD_NUMBER,$branch,$commitID
     $MSBuildConfiguration = 'Release'
+
+    $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 'true'
+}
+
+$minDotNetVersion = [version]'2.1.503'
+$dotnetVersion = $null
+$dotnetInstallDir = Join-Path -Path $PSScriptRoot -ChildPath '.dotnet'
+$dotnetPath = Join-Path -Path $dotnetInstallDir -ChildPath 'dotnet.exe'

```

Comment:
Windows specific logic that we'll have to fix later when we run a build on Linux/MacOS.

Reply:
Yes. I'll be fixing/addressing it over in `feature/cross-platform`. I was trying to be fast on this change.